generators/app/index.js

```diff
@@ -1,21 +1,72 @@
+const fs = require('fs');
 const Generator = require('yeoman-generator');
 const yosay = require('yosay');
 
+const scopePropsMap = new Map();
+scopePropsMap.set('@easyops', {
+  repository: 'Console-W',
+  subPackagePath: 'packages'
+});
+scopePropsMap.set('@plugin-common', {
+  repository: 'console-plugins',
+  subPackagePath: '@plugin-common'
+});
+const scopes = Array.from(scopePropsMap.keys());
+
 module.exports = class extends Generator {
   initializing() {
+    const done = this.async();
     this.log(yosay('Welcome to the Console Package generator!'));
     this.conflicter.force = true;
+    fs.readFile('package.json', {
```

Comment:
`const fs = require('fs').promises`

Reply:
experimental?

Reply:
It should be pretty stable from our perspective.